vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/client/legacy_client_migration_issue_23794_unittest.cc

```diff
@@ -19,7 +19,7 @@ namespace {
 
 constexpr char kClientIssue23794Filename[] = "client_issue_23794.json";
 constexpr uint64_t kClientIssue23794JsonHash = 1891112954;
-constexpr uint64_t kMigratedClientIssue23794JsonHash = 2655643370;
+constexpr uint64_t kMigratedClientIssue23794JsonHash = 1570672789;
```

Comment:
Changed because we removed `version_code` from the `client.json` and we need to regenerate the hash